src/Symfony/Component/HttpKernel/EventListener/CacheAttributeListener.php

```diff
@@ -163,6 +163,15 @@ public function onKernelResponse(ResponseEvent $event): void
             if (false === $cache->public) {
                 $response->setPrivate();
             }
+
+            if (true === $cache->noStore) {
```

Comment:
```suggestion
            if ($cache->noStore) {
```

Same for the `false` case below please

Reply:
For the false this wont work. The comparaisons are done in a loop (the attribute cascade targeting the controller).

We cannot test for "false" as cache->noStore has a null default value (to be honest i just used the previous lines of code as reference)

```php
        foreach ($attributes as $cache) {
            if (true === $cache->public) {
                $response->setPublic();
            }

            if (false === $cache->public) {
                $response->setPrivate();
            }

            if (true === $cache->noStore) {
                $response->setPrivate();
                $response->headers->addCacheControlDirective('no-store');
            }

            if (false === $cache->noStore) {
                $response->headers->removeCacheControlDirective('no-store');
            }
        }
```

What do you prefer?

* Keep it like that "for readability" / consistency ?
* Change only "my" addition (`true === cache->noStore`) ?
* Change also the `true === $cache->public` ?

?

Reply:
I missed the `null` case sorry, let's go as-is 👍